go/vt/mysqlctl/replication.go

```diff
@@ -276,6 +280,19 @@ func (mysqld *Mysqld) StartReplicationCommands(status *proto.ReplicationStatus)
 	return flavor.StartReplicationCommands(&params, status)
 }
 
+// SetMasterCommands starts a replication
```

Comment:
Looks like the comment needs to be updated, since it just returns the commands instead of actually starting.